chapter9.adoc

```diff
@@ -0,0 +1,408 @@
+
+* The RDSM configures `SRL` to 3 and `SSM` to 1 for the second SD and programs
+  the `srmcfg` with a value of 24. The second SD is allowed to program the low
+  order 3 bits of `srmcfg.RCID`, thereby selecting `RCID` values between 24 and 31.
+  When this SD reads `srmcfg.RCID`, the value returned consist of the low 3 bits,
+  with all upper bits set to 0.
+====
+
+The `RCID` and `MCID` values that accompany requests, when made with an effective
+privilege mode equal to M are stored in the `mrmcfg` register. Conversely, for
+requests made with effective privilege modes less than M, these values are
+stored in the `srmcfg` register.
+
+The Smqosid extension introduces a `MQRID` and a `SQRID` field in the `msdcfg`
+register to hold the `QRID` for requests made with effective privilege mode
+equal to M and for requests made with effective privilege mode less than M
+respectively. The corresponding QRID value accompanies such requests. The
+capacity and bandwidth controllers use the `QRID` along with `RCID` and `AT` to
+determine the resource allocation configurations. The capacity and bandwidth
+controllers use the `QRID` along with `MCID` to determine the ID of the
+counter used for resource usage monitoring.
+
+[[SDQRI]]
+=== Supervisor Domain QoS Register interfaces (QRI)
+
+Capacity and bandwidth controllers that support supervisor domains provide one
+or more memory-mapped QoS register interfaces (QRI). A QRI may be made
+exclusively accessible to a supervisor domain or the RDSM may mediate access to
+the QRI using an SBI. The RDSM can control access to the QRI from supervisor
+domains using MTT and/or PMP. The RDSM controls access to the QRI from devices
+using IOMTT and/or IOPMP.
+
+The number of `RCID` and `MCID` supported by the controllers for each QRI need
+not be identical. For maximal flexibility in allocation of `RCID` and `MCID`
+values, it is recommended that the number of `RCID` and `MCID` supported for a
+given `QRID` be identical in all capacity and bandwidth controllers in the
+system.
+
+The capacity and bandwidth controllers use the configurations that were
+established for the `RCID` and `AT` in the request through the QRI corresponding
+to the `QRID` in the request. Likewise the counters to count resource usage are
+selected using `MCID` and the `QRID` in the request and a QRI can be used to
+access counters associated with the corresponding `QRID`.
+
+By default all resources in the capacity and bandwidth controllers may be
+allocated using any of the QRI. The controllers may optionally support
+reservation of resources for use by a QRI. When such reservation is supported
+the capacity or bandwidth reserved for a QRI may only be used by requests that
+have the corresponding `QRID`. Supporting resource reservation capability allows
+effective partitioning of the shared resources among SDs sharing the resources.
+
+To optionally support capacity reservation to a QRI, the capacity controllers
```

Comment:
Suggestion: To reserve capacity to a QRI, the capacity controllers optionally support a new operation...

Reply:
Updated.